resources/Examples/compact_osw/files/python_plugin_program_erb.py

```diff
@@ -0,0 +1,44 @@
+from pyenergyplus.plugin import EnergyPlusPlugin
+
+import os, sys
+sys.path.append('C:/Python38/Lib/site-packages') # this should (needs to?) be same version as E+'s version
```

Comment:
We're going to have to make this path to system python's site_packages some kind of cmake replacement or a command line arg if we want to test this.

But more broadly, this is going to pose the question of whether we should/need to provide some common python packages out of the box like we do for ruby gems (and take into account the pain that it is to maintain that stuff). I'm specifically thinking about native packages (related to this issue https://github.com/NREL/OpenStudio/issues/4846), (pandas does have a C extension as well, I think something about numpy may be special)

@kbenne @tijcolem

Reply:
Following https://github.com/NREL/OpenStudio/pull/4868 you can make your test pass the `--python_home` or `--python_path` and it'll work. 

I think we should discuss whether we should always add some common python packages like Jinja2 (https://github.com/NREL/OpenStudio/issues/4906#issuecomment-1621966528)

Reply:
@jmarrec @kbenne 

Has there been any more discussion/movement on #4906?

Reply:
See here: https://github.com/NREL/OpenStudio/blob/0ab02805c62a5f459e669bf3f4c65c95a1d8eed7/src/cli/CMakeLists.txt#L343-L393